### DISCUSSION TITLE
[Bug]: about a few media players and firefox has no sound.

### DISCUSSION THREAD

**Discussion by Anonymous2716** [termux/termux-packages#12965] [Q&A] [answered]
### Problem description

Firefox plays local media or YouTube videos without Any audio.  

Vlc has this blue layer on top of  video ,probably decoder issue.
![vlc](https://user-images.githubusercontent.com/78570872/200749817-2df104e1-a312-4800-af9a-fdbc02de29fa.png)

mpv-x can't play webm video downloaded with yt-dlp (could be MPV specific issue):

```
?? mpv *webm
 (+) Video --vid=1 (*) (vp9 1920x1080 30.000fps)
 (+) Audio --aid=1 --alang=eng (*) (opus 2ch 48000Hz)
AO: [opensles] 48000Hz stereo 2ch float
[hwupload] no support for this hw format
[hwupload] hardware format not supported
[autoconvert] can't find video conversion for yuv420p
Cannot convert decoder/filter output to any format supported by the output.
Could not initialize video chain.
Video: no video
A: 00:00:20 / 00:35:32 (1%)

Exiting... (Quit)
```
All of the media players plays video and audio fine execpt vlc sometimes stops playing audio and Firefox doesn't have audio at all.

### What steps will reproduce the bug?

Play videos in firefox.

Play video in vlc, vlc-qt.

play webm format videos in MPV.

### System information

termux-info:

```
Termux Variables:
TERMUX_API_VERSION=0.50.1
TERMUX_APK_RELEASE=UNKNOWN
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_APP_PID=5539
TERMUX_IS_DEBUGGABLE_BUILD=0
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.118.0
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://packages.termux.dev/apt/termux-main stable main
# x11-repo (sources.list.d/x11.list)
deb https://packages.termux.dev/apt/termux-x11 x11 main
Updatable packages:
All packages up to date
termux-tools version:
1.31.0
Android version:
9
Kernel build information:
Linux localhost 4.4.146 #2 SMP PREEMPT Wed Sep 9 18:47:45 CST 2020 aarch64 Android
```

**Comment by xtkoba** [comment]
Please file separate issue per package.

**Comment by Anonymous2716** [comment]
@xtkoba currently opened one https://github.com/termux/termux-packages/issues/12964  after this one . should I keep that open ?

**Reply by xtkoba** [reply]
Please keep going. Let's just make this discussion marked as answered after every issue has been filed.

**Reply by Anonymous2716** [reply]
Okay thanks. @xtkoba

**Comment by xtkoba** [comment, marked as answer]
As for `mpv`, see https://github.com/termux/termux-packages/issues/5057#issuecomment-601627254.

The `firefox` part has a separate issue: #12964. I suppose `vlc` issue has already been resolved.

**Reply by Anonymous2716** [reply]
Then I guess no need to open issues for vlc and MPV.  Great.
